.github/workflows/ci.yml

```diff
@@ -16,6 +16,23 @@ jobs:
       - run: cargo build --workspace
       - run: cargo test --workspace --no-fail-fast
 
+  fuzz:
+    name: Build fuzzers
+    runs-on: ubuntu-latest
+    env:
+      # TODO: Remove this override once nightly doesn't have
+      # false positives on dead_code.
+      RUSTFLAGS: ""
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
I think we should pin to a specific nightly to prevent fluctuation with nightly bugs.

Reply:
Done